src/pkg/notification/config/config.go

```diff
@@ -0,0 +1,15 @@
+package config
```

Comment:
If this Configuration struct is used in notification pkg only, why need the `notification/config` pkg?

Reply:
Let's move it and do not export it.